advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -17,7 +17,8 @@ It's important to understand that while the WCAG can be incredibly helpful for b
 
 ### The Four Principles
 
-The WCAG are organized around four, core principles that should be kept in mind when implementing any sort of accessibility feature:
+The WCAG are organized around four, core principles (POUR) that should be kept in mind when implementing any sort of accessibility feature:
+You can use the acronym POUR for this, which stands for:
```

Comment:
Sorry, I missed this. Now that we have it in the introductory text, can this be removed please?

```suggestion
```